homeassistant/components/broadlink/sensor.py

```diff
@@ -119,8 +128,9 @@ def _update(self, retry=3):
             if data is not None:
                 self.data = self._schema(data)
                 return
-        except socket.timeout as error:
+        except (socket.timeout, OSError) as error:
```

Comment:
I mean socket.error is an abstraction from OSError. Maybe you can only catch OSError and that work also with socket errors.